### ISSUE TITLE
Cobaya implementation loads incorrect likelihoods

### ISSUE THREAD

**Issue by gplynch619** [AarhusCosmology/connect_public#28]
Currently, the cobaya implementation loads the incorrect likelihoods during iterative sampling. Instead of TTTEEE the TT likelihood gets loaded, and the low_l TT likelihood gets loaded in place of the EE likelihood.  `run_cobaya_iteration.py` should have the following changes to rectify this. At line 35:

```python
lkls = {'Planck_highl_TTTEEE_lite': {'name': 'planck_2018_highl_plik.TTTEEE_lite',
                                     'clik': os.path.join(path_clik, 'hi_l/plik_lite'),
                                     'path': os.path.join(path['cosmo'], 'cobaya/custom_likelihoods')},
        'Planck_lowl_TT':           {'name': 'planck_2018_lowl.TT',
                                     'clik': os.path.join(path_clik, 'low_l/commander'),
                                     'path': os.path.join(path['cosmo'], 'cobaya/custom_likelihoods')},
        'Planck_lowl_EE':           {'name': 'planck_2018_lowl.EE',
                                     'clik': os.path.join(path_clik, 'low_l/simall'), ## CHANGED low_l/commander -> low_l/simall
                                     'path': os.path.join(path['cosmo'], 'cobaya/custom_likelihoods')}}
```
And in the loop starting at line 62:
```python
for lkl in param.sampling_likelihoods:
    if lkl == 'Planck_lite':
        lkl = 'Planck_highl_TTTEEE_lite'
    if lkl in lkls:
        for name in os.listdir(lkls[lkl]['clik']):
            if 'TTTEEE' in lkl and name.endswith('TTTEEE.clik'):
                clik_file = os.path.join(lkls[lkl]['clik'], name)
                break ## ADDED, stops after finding TTTEEE
            elif name.endswith('.clik'):
                clik_file = os.path.join(lkls[lkl]['clik'], name)
                break ## ADDED, or stops after finding other file (TT)
        info['likelihood'][lkls[lkl]['name']] = {'clik_file':   clik_file,
                                                 'python_path': lkls[lkl]['path']}
    else:
        raise NotImplementedError(f"For now, only the following three likelihoods are available during training:\n{' '*4}Planck_highl_TTTEEE_lit, Planck_lowl_TT, Planck_lowl_EE\nYou can manually add extra likelihoods as a nested dictionary using Cobaya syntax in the parameter file, e.g.\n{' '*4}"+"extra_cobaya_lkls = {'Likelihood_name': {'path': path/to/likelihood,\n"+f"{' '*52}'options': other_options,\n{' '*52}"+"...},\n"+f"{' '*44}"+"...}")
```
I can submit a pull request later on, just need to clean some code up.

**Comment by AndreasNygaard** [comment]
Thanks again for the effort in finding all of these little bugs - and for the suggested solutions. There are a few things I need to sort out with the new Cobaya implementation, so I think I will do them all at once in the near future. Your code snippets are very useful for this and much appreciated.

**Comment by AndreasNygaard** [comment]
Resolved by #33
